src/main/java/software/amazon/qldb/QldbSession.java

```diff
@@ -128,6 +128,9 @@ boolean isClosed() {
                 if (executionContext.retryAttempts() >= retryPolicy.maxRetries()) {
                     throw sce;
                 }
+            } catch (Exception e) {
```

Comment:
We can remove L98-100

```
            } catch (TransactionAbortedException ae) {
                noThrowAbort(transaction);
                throw ae;
```

Reply:
This can still be thrown from the transaction executor and this line will end up doing the actual abort - keeping.

Reply:
Removed.